File: problem_solving_for_beginners/array_problems/python/count_even_and_odd_number.py
# Count even and odd number in the element of array 
arr = []
size = int(input("Enter size of array: "))
even_number_count = odd_number_count = 0

for i in range(size):
    i = int(input("Enter Element of array: "))
    arr.append(i)
    
#write your code here
for i in arr:
    if i % 2 == 0:
        even_number_count += 1
    else:
        odd_number_count += 1
        
print(f"Even Number Count = {even_number_count:d}")
print(f"Odd Number Count = {odd_number_count:d}")
    

File: problem_solving_for_beginners/array_problems/c-programming/count_even_and_odd_number.c
#include<stdio.h>

/**
 * main - Count even and odd number in the array
 * 
 * Return: 0 Always (success)
 */

int main(void)
{
    int arr[10], size, i;
    int even_number_count = 0, odd_number_count = 0;

    printf("Enter size of array: ");
    scanf("%d", &size);

    for(i = 0; i < size; i++)
    {
        printf("Enter Element of array: ");
         scanf("%d", &arr[i]);
    }
       
    //Write your code here
    for (i = 0; i < size; i++)
    {
        if (arr[i] % 2 == 0)
            even_number_count++;
        else
            odd_number_count++;
    }
    printf("Even Number Count = %d\n", even_number_count);
    printf("Odd Number Count = %d\n", odd_number_count);

    return 0;
}